src/clients/jobManagerWrapper.ts

```diff
@@ -53,14 +45,17 @@ export class JobManagerWrapper extends JobManagerClient {
     const expirationDate = new Date();
     expirationDate.setDate(expirationDate.getDate() + this.expirationDays);
 
+    const traceContext: ITraceParentContext = {};
+    propagation.inject(context.active(), traceContext);
+
     const jobParameters: IJobExportParameters = {
       roi: data.roi,
       callbacks: data.callbacks,
       crs: data.crs,
-      exportVersion: ExportVersion.ROI,
       fileNamesTemplates: data.fileNamesTemplates,
       relativeDirectoryPath: data.relativeDirectoryPath,
       gpkgEstimatedSize: data.gpkgEstimatedSize,
+      traceParentContext: traceContext,
```

Comment:
should be in same conetxt as overseer.
 was it in overseer in the job or the task? is it the same and the same name?